react/features/toolbox/components/web/Toolbox.js

```diff
@@ -980,6 +981,9 @@ class Toolbox extends Component<Props, State> {
                     key = 'fullscreen'
                     onClick = { this._onToolbarToggleFullScreen }
                     text = { _fullScreen ? t('toolbar.exitFullScreen') : t('toolbar.enterFullScreen') } />,
+            this.props._isModerator && <LiveStreamButton
```

Comment:
it's already used as variable with destructuring assignment
https://github.com/aeternity/jitsi-meet/blob/7561cd7fd0ec6a74fba670cd8c40a7ac542bd384/react/features/toolbox/components/web/Toolbox.js#L957-L971

Reply:
Added

Reply:
what? it's already at `develop`